src/game/movement.rs

```diff
@@ -139,7 +139,7 @@ pub fn apply_movement(
             _ => 1.0,
         };
 
-        position.offset += movement.velocity * dt * roll_multi;
+        position.offset += movement.velocity * roll_multi;
         position.fix_offset_overflow();
```

Comment:
I don't think this is quite right - since it'll still break movement when frame rates fluctuate